apps/andi/lib/andi_web/live/ingestion_live_view/extract_steps/extract_auth_step_form.ex

```diff
@@ -36,36 +36,36 @@ defmodule AndiWeb.ExtractSteps.ExtractAuthStepForm do
 
             <div class="extract-auth-step-form__destination">
               <%= label(f, :destination, DisplayNames.get(:destination), class: "label label--required", for: "step_#{@id}__auth_destination") %>
-              <%= text_input(f, :destination, [id: "step_#{@id}__auth_destination", class: "input", required: true]) %>
+              <%= text_input(f, :destination, [id: "step_#{@id}__auth_destination", aria_label: "step_#{@id}__auth_destination", class: "input", required: true]) %>
```

Comment:
I'm not sold on these aria_labels. They make the input have a unique label that's identifiable by a screen reader, but I'm worried the id is not going to make sense since it will probably look something like this: "step_44404bee-00ab-4f8b-a72b-3ae3b551ec95__auth_destination"

I'd be happy to try any alternatives.

Reply:
What is that ID? The process id? The component id?

Maybe the parent component could pass in a number indicating which order it was generated in / placed in, and that could be used instead?

Reply:
I thought about that, just need to figure out how to get an index from the elixir for loop render

Reply:
should be good here now